test/__snapshots__/deleteOriginalAssets.test.js.snap.webpack5

```diff
@@ -1,5 +1,34 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`"deleteOriginalAssets" option should delete original assets and keep source maps with option 'keep-source-map': assets 1`] = `
+Array [
+  Array [
+    "09a1a1112c577c2794359715edfcb5ac.png",
+    78117,
+  ],
+  Array [
+    "23fc1d3ac606d117e05a140e0de79806.svg",
+    393,
+  ],
+  Array [
+    "async.async.1c2911c7c819fc826fb7.js",
```

Comment:
hm, I think we should have source map here

Reply:
Source map was not generated for that.

```js
  [
      '09a1a1112c577c2794359715edfcb5ac.png',
      '23fc1d3ac606d117e05a140e0de79806.svg',
      'main.6a5afd3ff5b2f0fa8aaa.js',
      'async.async.1c2911c7c819fc826fb7.js',
      'main.6a5afd3ff5b2f0fa8aaa.js.map'
    ]
```

these are the total assets included for compression.